common/tools/dev-tool/src/commands/run/testNodeTSInput.ts

```diff
@@ -31,7 +32,9 @@ export default leafCommand(commandInfo, async (options) => {
     ? updatedArgs?.join(" ")
     : '--timeout 1200000 --exclude "test/**/browser/*.spec.ts" "test/**/*.spec.ts"';
   const command = {
-    command: `mocha ${defaultMochaArgs} ${mochaArgs}`,
+    command: isModuleProj
+      ? `mocha ${defaultMochaArgs} ${mochaArgs}`
+      : `cross-env TS_NODE_COMPILER_OPTIONS='{"module":"commonjs"}' mocha ${defaultMochaArgs} ${mochaArgs}`,
```

Comment:
Would this mess with the debugger?

Reply:
All should still be the same. All child processes are running under debugger as far as I can see.